geniza/corpus/forms.py

```diff
@@ -374,3 +376,40 @@ def clean(self):
         if rationale == "other" and not rationale_notes:
             msg = 'Additional information is required when selecting "Other".'
             self.add_error("rationale", msg)
+
+
+class TagChoiceField(forms.ModelChoiceField):
+    """Add a count of tagged documents to a tag label on a form (used for tag merging)"""
+
+    def label_from_instance(self, tag):
+        return "%s (%s tagged)" % (tag.name, tag.item_count)
+
+
+class TagMergeForm(forms.Form):
+    primary_tag = TagChoiceField(
+        label="Select primary tag",
+        queryset=None,
+        help_text=(
+            "Select the primary tag, which the other selected tags will be merged into. "
```

Comment:
not sure this wording makes sense here like it does for documents; it's really more about renaming, right? (but this is a quibble, you can get input from the team on what makes sense to them)